src/python/pants/backend/python/lint/black/rules_integration_test.py

```diff
@@ -73,7 +73,7 @@ def run_black(
         fmt_result = self.request_single_product(
             FmtResult,
             Params(
-                BlackFieldSets(field_sets, prior_formatter_result=input_snapshot),
+                BlackFieldSets(field_sets, prior_formatter_result=input_sources.snapshot),
```

Comment:
These lint integration test fixes are concerning. Test run green before and after. I stopped my yak shave here after a bit of investigation, but seems worth sussing out what's likely not being tested despite expectations.